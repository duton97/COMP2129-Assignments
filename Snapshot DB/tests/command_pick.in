SET a 1 2 3
PICK a 1
GET a
PICK a 3
PICK a 0
PICK b 0
BYE
