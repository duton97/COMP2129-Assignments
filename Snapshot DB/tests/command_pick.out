> ok

> 1

> [1 2 3]

> 3

> index out of range

> no such key

> bye
